gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.5.1-all.zip
-distributionSha256Sum=6c4276f97c9059ef4dfb06cc7325c43ed9b933ab04f032e9d9372e8589cb6faf
```

Comment:
Dont need this?

Reply:
Nice to have, but I got tired of adding it manually on every version up. Perhaps it's enough to use HTTPS?

Reply:
Yes I think so :)